app/src/lib/stores/helpers/repository-indicator-updater.ts

```diff
@@ -125,7 +125,7 @@ export class RepositoryIndicatorUpdater {
 
   private clearRefreshTimeout() {
     if (this.refreshTimeoutId !== null) {
-      window.clearTimeout()
+      window.clearTimeout(this.refreshTimeoutId)
```

Comment:
Somehow this wasn't a TS error before

Reply:
Woah! This is pretty terrible. I feel like we had an issue describing a case when the repository updater was running even though it was turned off in preferences or at least and observation. Thanks a bunch!